node/src/components/contract_runtime/operations.rs

```diff
@@ -161,6 +205,63 @@ pub fn execute_finalized_block(
                         .cloned()
                 },
             );
+
+    // Purging
+
+    match (activation_point, activation_point_block_height) {
+        (ActivationPoint::EraId(activation_point), Some(activation_point_block_height)) => {
+            let previous_block_height = finalized_block.height() - 1;
+
+            match calculate_purge_eras(
+                activation_point,
+                activation_point_block_height,
+                previous_block_height,
+                purge_batch_size,
+            ) {
+                Some(keys_to_purge) => {
+                    let first_key = keys_to_purge.first().copied();
+                    let last_key = keys_to_purge.last().copied();
+                    info!(previous_block_height, %activation_point_block_height, %state_root_hash, first_key=?first_key, last_key=?last_key, "commit_purge: preparing purge config");
```

Comment:
Can we help out rustfmt and ensure the log lines in this code block aren't over 100 chars pls?

Reply:
Done